packages/graph/hash_graph/bin/hash_graph/src/main.rs

```diff
@@ -35,6 +38,22 @@ async fn main() -> Result<(), GraphError> {
             err
         })?;
 
+    #[cfg(debug_assertions)]
+    let account_id = AccountId::new(Uuid::nil());
+    #[cfg(not(debug_assertions))]
+    let account_id = AccountId::new(Uuid::new_v4());
+
+    datastore
+        .insert_account_id(account_id)
+        .change_context(GraphError)
+        .await
+        .map_err(|err| {
+            tracing::error!("{err:?}");
+            err
+        })?;
```

Comment:
I don't think we want to short-circuit here as the binary won't ever start after the first time. We should ignore the result here if it's the duplicate foreign key error

Reply:
That's a good point